cmd/metrics-v2.go

```diff
@@ -186,8 +188,32 @@ const (
 	total             MetricName = "total"
 	freeInodes        MetricName = "free_inodes"
 
-	failedCount     MetricName = "failed_count"
-	failedBytes     MetricName = "failed_bytes"
+	failedCount        MetricName = "last_minute_failed_count"
+	failedBytes        MetricName = "last_minute_failed_bytes"
+	currActiveWorkers  MetricName = "current_active_workers"
+	avgActiveWorkers   MetricName = "average_active_workers"
+	maxActiveWorkers   MetricName = "max_active_workers"
+	recentBacklogCount MetricName = "recent_backlog_count"
+	currInQueueCount   MetricName = "last_minute_queued_count"
+	currInQueueBytes   MetricName = "last_minute_queued_bytes"
+	receivedCount      MetricName = "received_count"
+	sentCount          MetricName = "sent_count"
+	currTransferRate   MetricName = "current_transfer_rate"
+	avgTransferRate    MetricName = "average_transfer_rate"
+	maxTransferRate    MetricName = "max_transfer_rate"
+	credentialErrors   MetricName = "credential_errors"
```

Comment:
Since these are updated please make sure to update the `docs/metrics/prometheus/grafana/minio-bucket.json` and also `docs/metrics/prometheus/list.md`

Reply:
+1

Reply:
updated prometheus metrics list. grafana updates are not part of this PR.